src/ethereum/crypto/blake2.py

```diff
@@ -0,0 +1,217 @@
+"""
+The Blake2 Implementation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+"""
+import struct
+from dataclasses import dataclass
+from typing import List, Tuple
+
+from ethereum.base_types import Uint
+
+
+def get_words_from_le_bytes(data: bytes, start: int, num_words: int) -> List:
```

Comment:
Updated. I used `Uint` instead of `Uint64` because the `Uint64.MAX_WORD` has been defined as `2**64 - 1` whereas blake2f uses `2**64` for some operations. This will create issues if U64 is used.